libs/components/indicators/package.json

```diff
@@ -27,6 +27,7 @@
     "@skyux/theme": "0.0.0-PLACEHOLDER"
   },
   "dependencies": {
+    "@skyux/icon": "0.0.0-PLACEHOLDER",
```

Comment:
Since `@skyux/icons` is a direct dependency of `@skyux/indicators` now, what do you think about exporting `@skyux/icon` for all of the icon types in `@skyux/indicators`? Basically, we'd gut the indicators' icon (and harness) so that we'd be sure all usage is coming from the new `@skyux/icon` library, regardless if the import statement uses `@skyux/indicators` or `@skyux/icon`.

```
// Both would import from @skyux/icon:
import { SkyIconModule } from '@skyux/indicators';
import { SkyIconModule } from '@skyux/icon';
```